packages/v4/src/content/design-guidelines/content/units-and-symbols.md

```diff
@@ -0,0 +1,102 @@
+| Use consistent units, and avoid mixing them.                                                                                                                               | Write "10 to 75 seconds" instead of "10 seconds to 1.25 minutes."                                                                                                                                       |
+| Include a space between numbers and units, except for percentages.                                                                                                         | Add a space to "75 kg" but not to "75%."                                                                                                                                                                                                             |
+| Display the full name of each unit unless space is limited.                                                                                                                | Write "6 seconds ago" instead of "6s ago."                                                                                                                                                                                                                                |
+| Don't make unit symbols plural.                                                                                                                                            | Write "60 cm," not "60 cms."                                                                                                                                                                                                                                              |
+| Don't punctuate unit symbols unless they end a sentence.                                                                                                                   | "60 cm doesn't end this sentence, so 'cm' isn't followed by a period."                                                                                                                                                                                                   |
+| Write SI unit symbols in lowercase, unless a symbol is named after a person or proper noun, then capitalize its first letter. | Write "W" for watt.  <br /><br />Write "Hz" for hertz. |
+| Don't mix symbols or apply mathematical operations to unit names.                                                                                                          | Write "kg/m<sup>3</sup>" instead of "kilogram/m<sup>3</sup>."  <br /><br />Write "kg · m<sup>-3</sup>" instead of "kilogram/cubic meter."  <br /><br />Use "kilogram per cubic meter" instead of "kg/cubic meter," "kilogram per meter<sup>3</sup>," or "kg per m<sup>3</sup>." |
```

Comment:
Using quotes for these examples started looking confusing here. Do you think italics might look better for all these examples? We use italics instead of quotes in the guide's Terminology examples, so it would align with that, too. It might be worth trying just to see how it looks.

Reply:
Yes, I think italics make it look less confusing!